wcivf/apps/elections/templates/elections/includes/_postal_votes.html

```diff
@@ -0,0 +1,101 @@
+{% load i18n %}
+{% load postcode_tags %}
+{% load humanize %}
+
+<li>
+  <details open class="ds-stack-smaller">
```

Comment:
I've made this always open for now
but I could make it always closed
or open before the application deadline and then closed after
Thoughts?

Reply:
Yeah, closed after the deadline is nice :+1: